spec/clean-spec.coffee

```diff
@@ -24,24 +24,35 @@ describe 'apm clean', ->
       response.sendfile path.join(__dirname, 'fixtures', 'SHASUMS256.txt')
     app.get '/tarball/test-module-1.0.0.tgz', (request, response) ->
       response.sendfile path.join(__dirname, 'fixtures', 'test-module-1.0.0.tgz')
-    server =  http.createServer(app)
-    server.listen(3000)
+    server = http.createServer(app)
 
-    atomHome = temp.mkdirSync('apm-home-dir-')
-    process.env.ATOM_HOME = atomHome
-    process.env.ATOM_ELECTRON_URL = "http://localhost:3000/node"
-    process.env.ATOM_ELECTRON_VERSION = 'v0.10.3'
+    live = false
+    server.listen 3000, '127.0.0.1', ->
+      console.log "Server started"
+      atomHome = temp.mkdirSync('apm-home-dir-')
+      process.env.ATOM_HOME = atomHome
+      process.env.ATOM_ELECTRON_URL = "http://localhost:3000/node"
+      process.env.ATOM_ELECTRON_VERSION = 'v0.10.3'
 
-    moduleDirectory = path.join(temp.mkdirSync('apm-test-module-'), 'test-module-with-dependencies')
-    wrench.copyDirSyncRecursive(path.join(__dirname, 'fixtures', 'test-module-with-dependencies'), moduleDirectory)
-    process.chdir(moduleDirectory)
+      moduleDirectory = path.join(temp.mkdirSync('apm-test-module-'), 'test-module-with-dependencies')
+      wrench.copyDirSyncRecursive(path.join(__dirname, 'fixtures', 'test-module-with-dependencies'), moduleDirectory)
+      process.chdir(moduleDirectory)
+      live = true
+    waitsFor -> live
 
   afterEach ->
```

Comment:
```coffee
waitsFor (done) ->
  server.close -> done()
```
*should* work.

Reply:
Oh, good catch.

Reply:
... Nope:

```
TypeError: done is not a function
  at Server.<anonymous> (/Users/smashwilson/src/atom/apm/spec/clean-spec.coffee:40:9)
  at Object.onceWrapper (events.js:313:30)
  at emitNone (events.js:106:13)
  at Server.emit (events.js:208:7)
  at emitListeningNT (net.js:1378:10)
  at _combinedTickCallback (internal/process/next_tick.js:135:11)
  at process._tickCallback (internal/process/next_tick.js:180:9)
```

I bet that's implemented in a different Jasmine helper library? I know I've seen a few floating around.